### PR TITLE
Subordinate series fade

### PR DESCRIPTION
**Pull request by huwshimi**
Handle fading services without matching series to subordinates. Working towards #1918. Locking/changin series will come in a followup.

### CODE REVIEW

File: jujugui/static/gui/src/app/views/topology/topology.js [hunk 1]
```diff
@@ -284,19 +284,6 @@ YUI.add('juju-topology', function(Y) {
     },
 
     /**
-     Show the menu for a given service
-
-     @method showMenu
-     @param {String} serviceId
-    */
-    showMenu: function(serviceId) {
```
Comment (huwshimi) [inline jujugui/static/gui/src/app/views/topology/topology.js]: Drive-by removal of unused code.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5695/
Test PASSed.

**Comment by huwshimi** [comment]
QA:
- use the prod charmstore
- add xenial/nrpe, precise/gunicorn, trusty/mysql and precise/python-django
- click the add relation button for xenial/nrpe, it should fade all but trusty/mysql (nrpe is multi-series with trusty)
- click the add relation button for trusty/mysql, it should fade all but xenial/nrpe
- click the add relation button for precise/gunicorn, it should fade all but precise/python-django
- click the add relation button for precise/python-django, it should fade all but precise/gunicorn

File: jujugui/static/gui/src/app/models/endpoints.js [hunk 2]
```diff
@@ -120,14 +144,25 @@ YUI.add('juju-endpoints', function(Y) {
     Y.each(db.services, function(tgt) {
```
Comment (hatched) [inline jujugui/static/gui/src/app/models/endpoints.js]: Could this be de-yui'd please

File: jujugui/static/gui/src/app/views/topology/topology.js (hunk 1 above)
Comment (hatched) [inline jujugui/static/gui/src/app/views/topology/topology.js]: Thanks!

**Comment by hatched** [comment]
Thanks @huwshimi This works awesome!

:+1: with a de-yui plz :) 
:ok:

**Comment by kadams54** [comment]
:+1: - thanks @huwshimi!

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5708/
Test PASSed.

**Comment by huwshimi** [comment]
Thanks @hatched @kadams54 :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge
